files/usr/share/jeos-firstboot/jeos-firstboot-dialogs

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```

Comment:
Not meant to be executed, so this is misleading

Reply:
Yes, put this one there for some reason I don't remember tho, I think was some problem while testing something, gonna remove.
If you get the chance to test run, that would be cool, but I've been test running at each commit just to be sure, while on that I'm looking a way to make some test automation to have at least a smoke test automated.

Reply:
> If you get the chance to test run, that would be cool, but I've been test running at each commit just to be sure,
> while on that I'm looking a way to make some test automation to have at least a smoke test automated.

If you set up a package + image build on OBS, I can do an openQA test run (though the openQA test for jeos-firstboot isn't that extensive currently).

Reply:
Martin already did a openQA run a few commits before, but indeed the tests are not really covering much.

Reply:
Shebang is still there